test/integration/kube_api_discovery_test.go

```diff
@@ -437,11 +437,14 @@ func TestGetAPIResourceList(t *testing.T) { //nolint:gocyclo // each t.Run is pr
 			}
 		}
 
+		// manually update this value whenever you add additional fields to an API resource and then run the generator
```

Comment:
Copyright year is not updated in this file.

Reply:
👍